### DISCUSSION TITLE
Confusing Fritzing draw

### DISCUSSION THREAD

**Discussion by janseta** [danieleborgo/OctoPrint-ShutdownButtonLEDBuzzer#7] [Ideas]
Hi, 
Congratulations for your work!

I write only because I think the draw is confusing....
It took me a time to understand that the blue cable is a common cable for the 3 elements (led, buzzer and button...)
At the first look, it seems there is a blue direct cables between buton and led, and other beteen gpio and buzzer....

I suggest to change the draw by something like this:
![No confuising Fritzing](https://user-images.githubusercontent.com/36029398/150675823-49ac3b6e-7db8-4be3-a9d5-d11c7dafdbca.jpg)

**Comment by danieleborgo** [comment]
Hi,
I think that also your design may have the same problem. What about writing _GND_ on the drawing? Could this be more clear?
Thank you for your feedback

**Comment by janseta** [comment]
maybe now is enough clear...
![second draw](https://user-images.githubusercontent.com/36029398/152060532-f3c8afea-d08a-4db7-af76-60575beddea9.jpg)

**Comment by danieleborgo** [comment]
Image updated

**Comment by janseta** [comment]
;-)
